.travis.yml

```diff
@@ -1,3 +1,29 @@
 language: node_js
+
 node_js:
   - "6"
+
+services:
+  - docker
+
+env:
+  GOPATH: ${TRAVIS_BUILD_DIR}/_workspace
+
+hosts:
+    - alice.cozy.io
+    - bob.cozy.io
+    - charlie.cozy.io
```

Comment:
And I think it would be safer to use something like `alice.cozy.local`. It avoids to ping a real cozy instance the day we forget to declare here a fouth host used in tests.